supplementary_style_guide/glossary_terms_conventions/product_conventions/partner-and-hybrid-cloud-infrastructure.adoc

```diff
@@ -0,0 +1,762 @@
+
+[discrete]
+[[logical-volume]]
+==== image:images/yes.png[yes] logical volume (noun)
+*Description*: A virtual, block storage device that a file system, database, or application can use. To create a logical volume, physical volumes are combined into a volume group (VG). This creates a pool of disk space out of which logical volumes (LVs) can be allocated. Note that Logical Volume Manager (LVM) is the device mapper framework that provides logical volume management for the Linux kernel.
+
+*Use it*: Yes
+
+
+[discrete]
+[[primary-node]]
+==== image:images/yes.png[yes] primary node (noun)
+*Description*: Generically, the primary note is the machine that contains or runs the main components of a product.
+
+Use "control plane node" for the OpenShift Container Platform node that manages other nodes in its Kubernetes cluster and schedules pods to run on nodes.
+
+Use "controller node" for the Ansible node from which jobs are executed.
+
+Use "source node" for the node in the active cluster when discussing geo-replication in Red Hat Gluster Storage.
+
+*Use it*: Yes
+
+*Incorrect forms*: master node
+
+[discrete]
+[[metadata]]
+==== image:images/yes.png[yes] metadata (noun)
+*Description*: Generically, data that provides information about one or more other pieces of data.
+
+*Use it*: Yes
+
+[discrete]
+[[migrate]]
+==== image:images/caution.png[with caution] migrate (verb)
+*Description*: To automatically move data from one location to another, usually to proactively or reactively improve cluster performance. OpenShift Container Platform uses "auto-migrate" or "automatically migrate" in this case.
+
+In virtualization, "live migration" refers specifically to a virtual machine being moved from one host to another while remaining operational.
+
+*Use it*: With caution - for clarity, use the full term first before shortening to "migrate".
+
+[discrete]
+[[multipath]]
+==== image:images/caution.png[with caution] multipath (noun)
+*Description*: Device Mapper Multipath Input Output (DM-MPIO), also known as DM Multipathing or multipath, is a Linux utility that enables multiple input/output paths between server nodes and storage arrays to be configured into a single device path to provide redundancy and improve performance.
+
+*Use it*: With caution - this term may be unclear on its own.
+
+[discrete]
+[[n-way-replication]]
+==== image:images/yes.png[yes] n-way replication (noun)
+*Description*: Generically, refers to data being copied to multiple storage devices so that *n* copies are available in the storage cluster as a whole. This term is generally only used in Red Hat Gluster Storage.
+
+*Use it*: Yes
+
+
+[discrete]
+[[namespace-hci]]
+==== image:images/caution.png[with caution] namespace (noun)
+*Description*: An abstract group or context, generally used to identify and isolate resources from users, processes, and other resources that do not share the namespace. In the context of OpenShift, the term "project" is preferred over "namespace".
```

Comment:
I'm not sure if the last sentence `In the context of OpenShift, the term "project" is preferred over "namespace".` is totally true. It's true that they are nearly the same thing, and project is OpenShift-specific. But there are times when you do need to discuss creating a namespace as opposed to a project. I'd suggest removing that sentence, or looking into it further before including.

Reply:
OpenShift's section probably defines this better than we can regardless; removing